app/bingey-ui/src/components/Watchlist/Watchlist.jsx

```diff
@@ -6,13 +6,13 @@ import { Paper, Grid, Typography, Button } from '@material-ui/core';
 import AddCircleOutlineIcon from '@material-ui/icons/AddCircleOutline';
 import AddIcon from '@material-ui/icons/Add';
 
+import TitleCard from 'components/Watchlist/TitleCard/TitleCard';
+
 const useStyles = makeStyles((theme) => ({
```

Comment:
Should we port styling to `.scss` as you were already playing with styling classes?

Reply:
nezinau kaip perkelt theme.spacing(2) dali i sccs, be to sita dalis rasyta tavo, nes cia watchlistas